### PR TITLE
Refresh circleci

### PR DESCRIPTION
**Pull request by arcondello**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -433,44 +442,45 @@ workflows:
...
+              python-version: *python-versions
             exclude:
-              - numpy-version: '1.20.2'
-                image: 3.6-jessie
-              - numpy-version: '1.18.5'
-                image: 3.9-buster
-              - numpy-version: '1.17.3'
-                image: 3.9-buster
+              - numpy-version: <1.21
+                image: 3.6.8
```
Comment (hhtong) [inline .circleci/config.yml:466]: instead of `image`, should this be `python-version` instead?

Comment (arcondello) [reply .circleci/config.yml:466]: it should. Huh. Wonder how that's working.

Comment (arcondello) [reply .circleci/config.yml:466]: Interesting, it's ignoring the `--only-binary`. I'll take a look to see what's going on.

Comment (arcondello) [reply .circleci/config.yml]: for posterity, needed `--only-binary=:all:` rather than `--only-binary=all`

**Pull request merged**